scripts/spell_check/check_spelling.sh

```diff
@@ -62,23 +62,14 @@ done
 shift $((OPTIND - 1))
 
 # check if file list is provided
-# * from a pipe or from a file
 # * from ALL_CHANGED_FILES env var
 # * from command line parameters
-if [ -p /dev/stdin ] || [ ! -t 0 ]; then
-  # read from pipe input or file
-  read SCRIPT_INPUT
-  if [ -z "$SCRIPT_INPUT" ]; then
-    exit 0
-  fi
+if [ -z "$ALL_CHANGED_FILES" ]; then
+  # read from command line parameters
+  SCRIPT_INPUT="$@"
 else
-  if [ -z "$ALL_CHANGED_FILES" ]; then
-    # read from command line parameters
-    SCRIPT_INPUT="$@"
-  else
-    # read from env var
-    SCRIPT_INPUT="$ALL_CHANGED_FILES"
-  fi
```

Comment:
These options can be useful when running this script locally.

Reply:
I can't get the script to work on GitHub with them. Any ideas? Otherwise we should kill them to resurrect the workflow

Reply:
on my laptop running this cmd  works fine 
```bash
ALL_CHANGED_FILES="src/app/qgisapp.cpp src/app/qgisapp.h" scripts/spell_check/check_spelling.sh
``` 
or 
```bash
export ALL_CHANGED_FILES="src/app/qgisapp.cpp src/app/qgisapp.h" 
scripts/spell_check/check_spelling.sh
``` 
May be the action does not fill the env var as expected? I wanted to fix the " too long argument line" error that occurred with big PR. Reverting my changes will bring back this previous issue. To test this action I should have created another test project...

Reply:
The env var is correctly populated, but the portion of the script that detects for stdin / piped input always kicks in when run as part of the workflow, so the env var is not used.

Reply:
ok that strange it works locally but not on github...